persistence-test/src/main/java/io/stargate/it/PersistenceTest.java

```diff
@@ -196,15 +196,15 @@ public void testKeyspace() throws ExecutionException, InterruptedException {
   @Test
   public void testAlterAndDrop() throws ExecutionException, InterruptedException {
     createKeyspace();
-    ResultSet createTable =
-        dataStore
-            .queryBuilder()
-            .create()
-            .table(keyspace, table)
-            .column("created", Boolean, PartitionKey)
-            .build()
-            .execute()
-            .join();
+    //    ResultSet createTable =
```

Comment:
nit: Remove this commented-out variable?

Reply:
The reason I left it (and other occurrences in this class) is that there's other already commented code that references it:
```java
    // TODO: [doug] 2020-07-10, Fri, 16:50 More schemaAgreement to revisit
    // assertThat(createTable.waitedForSchemaAgreement()).isTrue();
```
So if someone were to uncomment that code in the future, they might not know what `createTable` is supposed to be.

The larger debate is whether we should have commented code at all, but this felt a bit off topic for this PR.

Reply:
Ah, good point... I missed that :facepalm: